src/test/quick-start/explore-pipeline-quickstart.js

```diff
@@ -0,0 +1,58 @@
+export const explorePipelinesQuickStart = {
```

Comment:
in this example the quick start files are JS objects. Alternatively they could be yaml files, in that case the app build has to process them to convert them to json.